config/default.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
   jwtconf: {
     secret: 'supersecret',
     time: {
-      expiresIn: 86400,
+      expiresIn: 8640000000000,
```

Comment:
Write the format in a more friendly manner as  30 * 3600 * 1000 // 30 hours , or put in a comment how much time is it